server/tests/unit/common/apis/test_api_v3.py

```diff
@@ -726,6 +726,17 @@ def test_post_summaryvar_lossy(self):
                 self.assertEqual(df["col_idx"], [query_hash])
                 self.assertAlmostEqual(df["columns"][0][0], -0.17065382)
 
+    def test_uns_metadata_get(self):
+        endpoint = "uns/meta"
+        query = "key=spatial"
+        for url_base in [self.TEST_URL_BASE, self.TEST_URL_BASE_SPARSE]:
+            with self.subTest(url_base=url_base):
+                url = f"{url_base}{endpoint}?{query}"
+                header = {"Accept": "application/ojson"}
+                result = self.client.get(url, headers=header)
+                self.assertEqual(result.status_code, HTTPStatus.OK)
+                self.assertEqual(result.headers["Content-Type"], "application/json")
```

Comment:
should we test the actual response content matches expected metadata?

Reply:
yes we should